### PR TITLE
Upgrade to Kafka 2.2.0

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Enhancement / new feature

### Description

Add support for Kafka 2.2.0.

I guess the big question is ... should we drop 2.0.x?

### Checklist

- [x] Make sure all tests pass
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [x] Update CHANGELOG.md

### CODE REVIEW

**Comment by scholzj** [comment]
@strimzi-ci run tests tags=acceptance,regression

File: documentation/book/snip-images.adoc [hunk 1]
```diff
@@ -8,15 +8,16 @@ a|
 * {DockerOrg}/kafka:{DockerTag}-kafka-2.0.1
 * {DockerOrg}/kafka:{DockerTag}-kafka-2.1.0
 * {DockerOrg}/kafka:{DockerTag}-kafka-2.1.1
+* {DockerOrg}/kafka:{DockerTag}-kafka-2.2.0
 
 a|
-{ProductName} image for running Kafka, including:
+AMQ Streams image for running Kafka, including:
```
Comment (ppatierno) [inline documentation/book/snip-images.adoc]: Why AMQ Streams here?

Comment (scholzj) [reply documentation/book/snip-images.adoc]: Good that you noticed this. I didnt changed that manually - there seems to be bug in one of the scripts generating this. I fixed it now (in the script + regenerated the docs).

File: HACKING.md [hunk 2]
```diff
@@ -235,4 +235,4 @@ However, since system tests use command line `Executor` for some actions, make s
 System tests uses admin user for some actions. You can specify admin user via variable `TEST_CLUSTER_ADMIN` (by default it use `developer` because `system:admin` cannot be used over remote connections).
 
 ### Execute ST with custom Kafka version
-To set custom Kafka version in system tests need to add system property `ST_KAFKA_VERSION` with one of the following values: `2.0.0`, `2.0.1`, `2.1.0`
\ No newline at end of file
+To set custom Kafka version in system tests need to add system property `ST_KAFKA_VERSION` with one of the following values: `2.0.0`, `2.0.1`, `2.1.0`, `2.1.1` or `2.2.0`
```
Comment (tombentley) [inline HACKING.md]:
```suggestion
To set custom Kafka version in system tests need to add system property `ST_KAFKA_VERSION` with one of the versions of Kafka given in the `kafka-versions` file.
```

File: kafka-versions [hunk 3]
```diff
@@ -25,4 +25,5 @@
 2.0.0              2.0     2.0   b28e81705e30528f1abb6766e22dfe9dae50b1e1e93330c880928ff7a08e6b38ee71cbfc96ec14369b2dfd24293938702cab422173c8e01955a9d1746ae43f98
```
Comment (tombentley) [inline kafka-versions]: We have https://github.com/strimzi/strimzi-kafka-operator/issues/1426 to discuss, but I think we need to do something soon about this ever growing list.

Comment (scholzj) [reply kafka-versions]: Well, I asked about that above ... so ... what is your opinion?

**Comment by scholzj** [comment]
@strimzi-ci run tests tags=acceptance,regression

**Comment by scholzj** [comment]
@strimzi-ci run tests tags=acceptance,regression

**Pull request merged**